instat/dlgViewGraph.Designer.vb

```diff
@@ -178,7 +168,6 @@ Partial Class dlgViewGraph
     Friend WithEvents rdoDisplayOutputWindow As RadioButton
     Friend WithEvents ucrPnlDisplayOptions As UcrPanel
     Friend WithEvents rdoDisplayInteractiveView As RadioButton
-    Friend WithEvents lblRowFactor As Label

```

Comment:
Removed this lbl as it didn't have any text and there was no code for it in the dialog. I changed the tab order accordingly